manifests/bootstrap/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: bootstrap
+description: Bootstrap the app of apps
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
```

Comment:
Consider using `v0.0.1 # x-release-please-version`

Reply:
Do we need this here? I don't think this should follow the version of the repo considering we will release it as "app-of-apps.yaml"

Reply:
Right, but it's all about consistency. Especially since it is a bootstrap and it's still part of the project, IMHO it should be versioned.

Reply:
Sorry I forgot to comment here, if you look at the comment it says that "the version should be increased each time you make changes to the chart and its templates".
If we add that notation we are bumping the version on the repo release that breaks the comment proposition.

Reply:
Fine by me, I was just trying to align it to the other charts we host as well. All fine though :)